src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -835,6 +845,15 @@ public function run() {
     return 0;
   }
 
+  // Default value for $recursive = true so we do not break existing behavior
+  private function submoduleUpdate($repository_api, $recursive = true) {
+    $command = 'submodule update --init';
+    if($recursive)
+      $command .= ' --recursive';
```

Comment:
I did at first. But then reading http://php.net/manual/en/control-structures.if.php I noticed that they skip braces in their first example as well as in the description.